dynatrace/testing/api/settingstest.go

```diff
@@ -217,10 +217,6 @@ func TestAcc(t *testing.T, opts ...TestAccOptions) {
 		return
 	}
 
-	if v := os.Getenv("DYNATRACE_API_TOKEN"); v == "" {
```

Comment:
FYI, I also removed this limitation, as we can also run AC tests with just platform credentials